client/src/components/Settings/SettingsPatronModeBox/SettingsPatronModeBox.module.scss

```diff
@@ -9,3 +9,11 @@
   justify-content: center;
   margin-left: 1.2rem;
 }
+
+.tooltipBox {
+  .tooltip {
+    @media #{$desktop-up} {
+      left: -13.6rem;
```

Comment:
Instead of adding custom styles, please expand `position` prop in `Tooltip` and move the styles there.

Reply:
It must be a custom style because it is centered relative to the container, not the hint question mark.